python/tvm/relay/frontend/keras.py

```diff
@@ -1102,7 +1178,12 @@ def _convert_input_layer(keras_layer):
 
     etab = ExprTable()
     # Set global data format.
-    assert layout in ["NCHW", "NHWC", "NDHWC"], "Layout must be one of 'NCHW', NHWC or NDHWC"
+    assert layout in [
+        "NWC",
+        "NCHW",
+        "NHWC",
+        "NDHWC",
+    ], "Layout must be one of 'NWC', 'NCHW', NHWC or NDHWC"
```

Comment:
looking at this made me realize we should just have 'channels_first' and 'channels_last' as layout options so that it applies to all dimensions. No change needed for this PR but we should keep it in mind as a backlog item to fix.